cameraServer.py

```diff
@@ -56,14 +56,18 @@ def startCamera(config):
 
 
 #Process Functions
-def getRetroPos(img, display=False, sample=False):
-    """Function for finding retro-reflective tape a"""
+def getRetroPos(img, display=False, distance_away=distance_away):
+    """Function for finding retro-reflective tape"""
+    '''
+    newimg = img[:,:,1].astype(np.int32) - img[:,:,2] - img[:,:,0]
+    newimg -= newimg.min()
+    newimg = newimg.astype(np.float64)*255/newimg.max()
+    newimg = np.int0(newimg).astype(np.uint8)
+    '''
```

Comment:
What is this?  A comment, docstring?  If it's not being used you can get rid of it.